portal-web/test/functional/com/liferay/portalweb/macros/RemoteApps.macro

```diff
@@ -34,6 +35,66 @@ definition {
 		PortletEntry.publish();
 	}
 
+	macro addCustomElementAllFields {
+		LexiconEntry.gotoAdd();
+
+		Select(
+				locator1 = "Select#TYPE",
+				value1 = "Custom Element");
+
+		RemoteAppsEntry.addCustomElementAllFields(
+				entryName = "${entryName}",
+				entryHTMLName = "${entryHTMLName}",
+				entryURL = "${entryURL}",
+				entryCSSURL = "${entryCSSURL}",
+				entryProperties = "${entryProperties}");
+
+		Click(locator1 = "RemoteAppsEntry#REMOTE_APPS_INSTANCEABLE");
+
+		PortletEntry.publish();
+	}
+
+	macro editCustomElement {
+
+		TripleClick(
+				key_text = "Name",
+				locator1 = "TextInput#ANY");
```

Comment:
Type cannot be used to edit the field of the form because it just add the new text to the previous one.
So triple click is the solution I found to select the previous text and overwrite it typing the new one.  (Triple click because it can select all the text, even if there are spaces between words).